config/routes.rb

```diff
@@ -9,4 +9,7 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources  :users do
+    resources :posts
+  end
```

Comment:
- [ ] Kindly make sure to include only the 4 routes, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.) requirement. You can do something like the following to do so:
```ruby 
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thanks for the hint on how to make my code better. The best review I have received in a long time. Keep it up 👍 😄